Tools/scripts/run_tests.py

```diff
@@ -25,7 +25,7 @@ def main(regrtest_args):
             '-u',                 # Unbuffered stdout and stderr
             '-W', 'default',      # Warnings set to 'default'
             '-bb',                # Warnings about bytes/bytearray
-            '-E',                 # Ignore environment variables
+            #'-E',                 # Ignore environment variables
```

Comment:
Instead of removing -E, could we pass down the flag (-p new) if the parent has the flag or sees the environment variable?

Alternatively, we could just remove -E if the parent sees the environment variable, but in general I would recommend against just remove -E all the time as this was there to prevent errors due to environment interference in tests.